src/public/app/widgets/buttons/global_menu.js

```diff
@@ -333,7 +333,6 @@ export default class GlobalMenuWidget extends BasicWidget {
 
         const latestVersion = await this.fetchLatestVersion();
         this.updateAvailableWidget.updateVersionStatus(latestVersion);
-        this.$updateToLatestVersionButton.toggle(latestVersion > glob.triliumVersion);
```

Comment:
Since the widget was already toggling itself, I didn't see why this was necessary. (obviously it still worked after removing this line)

Reply:
@perfectra1n , can you double-check, please? If I change the version in `package.json` to `0.90.7-beta`, I get the update available badge but no menu entry:

![image](https://github.com/user-attachments/assets/249c53c7-6742-40f6-8012-8e7646a5c5d9)

If I put back this piece of code I get this:

![image](https://github.com/user-attachments/assets/59bdcff4-36cd-4709-af95-9db4d0bfc963)

Reply:
Oh so THAT’S what that was for ☠️

Reply:
Fixed, tested with the string `v0.90.7-beta`